PINCache/PINDiskCache.m

```diff
@@ -116,6 +128,20 @@ - (instancetype)initWithName:(NSString *)name rootPath:(NSString *)rootPath
     return self;
 }
 
+- (instancetype)initWithName:(NSString *)name rootPath:(NSString *)rootPath serializer:(PINDiskCacheSerializerBlock)serializer deserializer:(PINDiskCacheDeserializerBlock)deserializer
```

Comment:
Do you mind switching the implementation of this and `initWithName:(NSString *)name rootPath:` ? Then have initWithName:(NSString *)name rootPath call this one with the default serializer and deserializer?